_functions_tomcat.sh

```diff
@@ -151,32 +151,23 @@ do_configure_tomcat_jod() {
 }
 
 do_configure_tomcat_ldap() {
-  if [ -e ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} ]; then
-    # Reconfigure $DEPLOYMENT_GATEIN_CONF_PATH
-
-    # Ensure the configuration.properties doesn't have some windows end line characters
-    # '\015' is Ctrl+V Ctrl+M = ^M
-    cp ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.orig
-    tr -d '\015' < ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.orig > ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}
-
-    # Patch to reconfigure $DEPLOYMENT_GATEIN_CONF_PATH for ldap
-    find_instance_file LDAP_GATEIN_PATCH "${ETC_DIR}/gatein" "ldap-configuration.properties.patch" "${LDAP_GATEIN_PATCH_PRODUCT_NAME}"
-
-    # Reconfigure $DEPLOYMENT_GATEIN_CONF_PATH for LDAP
-    if [ "${LDAP_GATEIN_PATCH}" != "UNSET" ]; then
-      # Prepare the patch
-      cp ${LDAP_GATEIN_PATCH} ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.patch
-      echo_info "Applying on $DEPLOYMENT_GATEIN_CONF_PATH the patch $LDAP_GATEIN_PATCH ..."
-      cp ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.ori.ldap
-      patch -l -p0 ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} < ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.patch
-      cp ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH}.patched.ldap
-
-      replace_in_file ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} "@DEPLOYMENT_LDAP_URL@" "${DEPLOYMENT_LDAP_URL}"
-      replace_in_file ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} "@DEPLOYMENT_LDAP_ADMIN_DN@" "${DEPLOYMENT_LDAP_ADMIN_DN}"
-      replace_in_file ${DEPLOYMENT_DIR}/${DEPLOYMENT_GATEIN_CONF_PATH} "@DEPLOYMENT_LDAP_ADMIN_PWD@" "${DEPLOYMENT_LDAP_ADMIN_PWD}"
-      echo_info "Done."
+  configurable_env_var "DIRECTORY" ""
```

Comment:
Usually, variable initializations are on the main ``_functions.sh`` and persisted on the configuration file.

Perhaps something like ``USER_DIRECTORY`` or ``USER_DIRECTORY_TYPE`` could be less confusing